### DISCUSSION TITLE
Transcribe offline with GPU

### DISCUSSION THREAD

**Discussion by johnfelipe** [URUWorks/TeroSubtitler#204] [Announcements]
Is possible generate subtitle for audio or video offline with whisper using locally GPU?

**Comment by URUWorks** [comment]
It is possible with faster whisper. What operating system do you use?

**Comment by johnfelipe** [comment]
Windows, is possible use large V3, because I need extremely accuracy

**Comment by URUWorks** [comment]
correct, it is possible.

**Comment by johnfelipe** [comment]
Can u guide me for complete that task?

**Comment by URUWorks** [comment]
If you have already downloaded "FFmpeg" and "Faster-Whisper", then all that remains is to download the model to use. In this case, you mention "large-v3" which has not yet been updated in Tero Subtitler, so you have to install it manually. For this you download it from "https://huggingface.co/Purfview/" and copy it to the whisper folder, for example: "D:\Tero\Whisper\Models\faster-whisper-large-v3".

Otherwise you can use Tero and download V2 directly.

**Comment by johnfelipe** [comment]
is not working with GPU, how can solve that?
![SNAG-0161](https://github.com/URUWorks/TeroSubtitler/assets/428820/2c4def87-ad37-4673-a1d4-29aaf4921540)

**Reply by URUWorks** [reply]
GPU execution requires CUDA v11.x and [cuBLAS and cuDNN libs](https://github.com/Purfview/whisper-standalone-win/releases/tag/libs).

you got it ?

**Reply by Purfview** [reply]
> is not working with GPU

What error do you get?

**Reply by johnfelipe** [reply]
can u share me CUDA v11.x for windows 10 64 bits pls, for not shoot to another missed file

**Comment by johnfelipe** [comment]
i download and put in portable folder but is not taken GPU

![SNAG-0249](https://github.com/URUWorks/TeroSubtitler/assets/428820/9a4b2bbe-e423-463f-a07d-8ae2c42a0c30)

may be with your help i will use more TeroSubtitler
